util/schemautil/schemautil.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2018 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package schemautil
+
+import "github.com/pingcap/parser/model"
+
+// FindIndexByName finds index by name.
+func FindIndexByName(idxName string, indices []*model.IndexInfo) *model.IndexInfo {
```

Comment:
Why put this function here? It seems that this function only used in the package of ddl now.

Reply:
Also use in `planner/core/planbuilder.go`

Reply:
done.

Reply:
A better place is "github.com/pingcap/tidb/expression"